.travis.yml

```diff
@@ -0,0 +1,27 @@
+dist: jammy
+language: node_js
+node_js:
+  - lts/*
+stages:
+  - lint
+  - build
+jobs:
+  include:
+    - stage: lint
+      name: lint
+      script: npm run lint
+    - stage: lint
+      name: lint:css
+      script: npm run lint:css
+    - stage: build
+      name: build
+      script: npm run build
+      deploy:
+        provider: pages
+        skip_cleanup: true
```

Comment:
Travis says:

> <code>jobs.include.deploy</code>: deprecated key <code>skip_cleanup</code> (not supported in dpl v2, use cleanup)

Reply:
Although it's unclear if it means "you should change this" or "you will have to change this in order to upgrade"?

Reply:
I am getting confusing information online

https://github.com/travis-ci/dpl/issues/1138
https://travis-ci.community/t/misleading-dpl-v2-deprecation-warnings-with-v1-config/8153

Both _seem_ to indicate they had issues switching to `cleanup: false` while still on `dpl v1`? The first person ended up opting into dpl v2, the second ended up just ignoring the warning.

It seems dpl v2 is still in beta -> maybe just ignore like the second person? idk

Reply:
Yeah, I think that points to the second interpretation ("you will have to change this in order to switch to v2") and we should just ignore it.